### ISSUE TITLE
Random white flicker

### ISSUE THREAD

**Issue by AbdulmohsenAlotaibi** [jfoenixadmin/JFoenix#712]
Hi there,

I have been using jfoenix for a while now, and I started to notice white flickers when I started using JFXTreeTableView, I'm not sure if it is the cause, but it happens.

Basically after using the program for 3-10 minutes, everything Jfoenix related starts to flicker to white, when I focus/click on the object it appears normally.

I have searched and found this: https://github.com/jfoenixadmin/JFoenix/issues/646

I'm using Jfoenix for java 9.

Can you please suggest solutions so I can try them to see if it fixes the issue ?

Thank you.

**Comment by jfoenixadmin** [comment]
Hi,
were you able to re-produce the issue using the latest JFoenix version 9.0.4?

**Comment by AbdulmohsenAlotaibi** [comment]
The issue is that I cannot provide a walk through on how to produce this issue, because it just happens with no reason why.

Simply when I run the application, it works fine for a while, after a couple of restarts of the application it starts to flicker to white (only JFoenix components).

I have noticed that some components like jfxcheckbox are not affected, only jfxbutton, jfx tree (slider & checkbox inside it).

The version that i'm using is 9.0.4.

**Comment by jfoenixadmin** [comment]
Well, it's a kinda hard to reproduce the issue if no steps are provided.
are you reloading css files on your scene? can you reproduce the issue if css files are not used ?

**Comment by AbdulmohsenAlotaibi** [comment]
I have tried removing my css file but still it shows the same problem.

The thing with this bug is that it is very strange and might not happen after following some pattern.

But today I have found a new thing, I have noticed that when this bug is present, and I resize the window to the original anchorpane size, the bug is no longer there, and when I resize the window back and make it bigger, this bug happens.

So it seems resizing the window is affecting this.

I will try to reproduce this issue and then I'll upload the code.

I'll be using scene builder for fxml.
